myget.bat

```diff
@@ -0,0 +1,25 @@
+REM This is all ugly... we'll want to revisit this as soon as possible
+call dnvm use 1.0.0-rc1-update1 -r clr
+
+REM Temporary measure while we resolve oddities with the local feed
+REM cache and version numbers. Basically, we want to fetch these from
+REM nuget.org.
+rmdir /s /q c:\Users\Wonka\.dnx\packages\Google.Apis
+rmdir /s /q c:\Users\Wonka\.dnx\packages\Google.Apis.Core
+rmdir /s /q c:\Users\Wonka\.dnx\packages\Google.Apis.Auth
+
+call dnu restore
+IF ERRORLEVEL 1 EXIT /B 1
+
+call dnu build src\Google.Storage.V1
+IF ERRORLEVEL 1 EXIT /B 1
+call dnu build test\Google.Storage.V1.Tests
+IF ERRORLEVEL 1 EXIT /B 1
+call dnu build test\Google.Storage.V1.IntegrationTests
+IF ERRORLEVEL 1 EXIT /B 1
+call dnu build demo\Google.Storage.V1.Demo
+IF ERRORLEVEL 1 EXIT /B 1
+call dnx -p test\Google.Storage.V1.Tests test
+IF ERRORLEVEL 1 EXIT /B 1
+call dnu pack src\Google.Storage.V1
```

Comment:
Well, that would only build Google.Storage.V1 - it wouldn't build the tests or the demo.
I could pack that and then build the rest, but I think it's a bit simpler this way - and by being more explicit about each step, I expect it'll be simpler to diagnose errors.